chapters/revisions.tex

```diff
@@ -1196,7 +1196,7 @@ \subsection{Main changes in Modelica 3.2}\label{main-changes-in-modelica-3-2}
   Indian users (see grammar changes in \cref{lexical-conventions}). Modelica files are
   UTF-8 encoded, and can start with the UTF-8 encoded byte order mark
   (0xef 0xbb 0xbf) to indicate that it may contain UTF-8 characters;
-  this is treated as white-space in the grammar (see \cref{mapping-a-package-class-hierarchy-into-a-single-file-nonstructured-entity}).
+  this is treated as whitespace in the grammar (see \cref{mapping-a-package-class-hierarchy-into-a-single-file-nonstructured-entity}).
```

Comment:
This seem change (and the similar one in the other file) spurious, the variant with hyphen seem more common.
(Ten times more common according to google.)

Reply:
I just wanted it to be the same everywhere, and went with the variant that was more common in our document.  Now they are all _white-space_.